src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/ServiceBusConfiguration.java

```diff
@@ -1,25 +1,55 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.config;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.IMessageHandler;
+import com.microsoft.azure.servicebus.IQueueClient;
+import com.microsoft.azure.servicebus.MessageHandlerOptions;
 import com.microsoft.azure.servicebus.QueueClient;
 import com.microsoft.azure.servicebus.ReceiveMode;
 import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
 import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Lazy;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.ErrorNotificationService;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.MessageAutoCompletor;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.ServiceBusHelper;
+import uk.gov.hmcts.reform.bulkscanprocessor.tasks.ErrorNotificationHandler;
+
+import java.time.Duration;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import javax.annotation.PostConstruct;
 
 @Lazy
 @Configuration
 public class ServiceBusConfiguration {
 
+    private static final ExecutorService QUEUE_READ_EXEC = Executors.newSingleThreadExecutor(r ->
+        new Thread(r, "notifications-queue-read")
+    );
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private ErrorNotificationService errorNotificationService;
+
+    @Value("${queues.read-notifications.enabled}")
+    private boolean readNotificationsEnabled;
+
+    @Value("${queues.read-notifications.connection-string}")
+    private String connectionString;
+
+    @Value("${queues.read-notifications.queue-name}")
+    private String queueName;
```

Comment:
For `queueName` and `connectionString` it would be worth adding some prefix to make it clear these fields are related with notifications queue. This can be deducted from the annotations above declarations, but you'd have to navigate there to find out.

Reply:
fair point. can be amended in case other `@Value`s come into play. but it's not arm-breaking to include some prefix here